addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -125,6 +125,23 @@
                 <span t-field="channel.product_id.list_price" t-options="{'widget': 'monetary', 'display_currency': product_info['currency_id']}"/>
             </div>
         </div>
+        <t t-if="channel.prerequisite_channel_ids and not channel.prerequisite_user_has_completed">
```

Comment:
Note that this template ("course_buy_course_button") is also used in "course_option_buy_course_now".

If I remember well it's an option in the customize menu of the website.

Would be nice to re-test that it's compatible with your changes here if not already done.